cloudnet-driver/src/main/java/de/dytanic/cloudnet/driver/service/ServiceConfiguration.java

```diff
@@ -544,10 +673,18 @@ public Builder processParameters(Collection<String> jvmOptions) {
         }
 
         /**
-         * The jvm options for the new service. They will be added directly before the "-Xmx" parameter in the startup command.
+         * The process parameters for the new service. This will be the last parameters that will be added to the command.
          */
-        public Builder jvmOptions(String... jvmOptions) {
-            return this.jvmOptions(Arrays.asList(jvmOptions));
+        public Builder addProcessParameters(String... jvmOptions) {
+            return this.addProcessParameters(Arrays.asList(jvmOptions));
+        }
+
+        /**
+         * The process parameters for the new service. This will be the last parameters that will be added to the command.
+         */
+        public Builder addProcessParameters(Collection<String> jvmOptions) {
+            this.config.processConfig.processParameters.addAll(jvmOptions);
```

Comment:
Same here

Reply:
I don't think this is null because its initialized `protected Collection<String> processParameters = new ArrayList<>();`